bricksrc/setpoint.py

```diff
@@ -1,347 +1,261 @@
-from .namespaces import TAG, BRICK, RDFS, QUDTQK
```

Comment:
dont change the import path

Reply:
@jbkoh After moving setpoints into parameter, I found that QUDTQK wasn't imported. If this remains unchanged, we will need to remove the BRICK.hasQauntity properties

Reply:
don't understand what this means and the relevance between this line and hasQuantity

Reply:
For example, this was recently added to Parameter. I updated the namespace import to include QUDTQK:

```
                    "Humidity_Deadband": {
                        BRICK.hasQuantity: QUDTQK.RelativeHumidity,
```

Reply:
You can add more imports. My ask was not to modify the import path from `from .namespaces` to  `from namespaces`